src/components/teasers/teaser.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import {
+  TeaserTitle,
+  TeaserTitleLarge,
+  Text,
+  Timestamp,
+  Topline,
+} from '../typography/typography';
+import { Arrow } from '../icons/arrow';
+import { theme } from '../layout/theme';
+import { GatsbyImage, IGatsbyImageData } from 'gatsby-plugin-image';
+import { Link } from '../links/links';
+import { up } from '../breakpoint/breakpoint';
+
+const TeaserContainer = styled.div`
+  cursor: pointer;
+  overflow: hidden;
+
+  .teaser-img {
+    transition: transform 0.2s;
+  }
+
+  &:hover {
+    color: ${theme.palette.text.topline};
+    p {
+      color: ${theme.palette.text.topline};
+    }
+
+    .teaser-img {
+      transform: scale(1.05);
+    }
+  }
+`;
+
+const ToplineContainer = styled.div`
+  display: flex;
+  margin-top: 16px;
+  width: 100%;
+  justify-content: space-between;
+`;
+
+const StyledTeaserTitleLarge = styled(TeaserTitleLarge)<{
+  hideOverflow?: boolean;
+}>`
+  ${(props) =>
+    props.hideOverflow &&
+    css`
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+    `}
+`;
+const StyledTeaserTitle = styled(TeaserTitle)<{ hideOverflow?: boolean }>`
+  ${(props) =>
+    props.hideOverflow &&
+    css`
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+    `}
+`;
+
+const TeaserText = styled(Text)<{ hasImage: boolean; hideOverflow?: boolean }>`
+  margin-bottom: 22px;
+  margin-top: ${(props) => (props.hasImage ? '8px' : '16px')};
```

Comment:
the margin should be applied from the image, that way you can remove the supporting prop `hasImage`, can't you ?

Reply:
The image has an influence on the margin between headline and text, that's why I used this property.
<img width="306" alt="Bildschirmfoto 2021-11-16 um 10 02 50" src="https://user-images.githubusercontent.com/57712895/141955256-920edbfb-319b-419f-b22f-279e9c6ffb4f.png">
<img width="323" alt="Bildschirmfoto 2021-11-16 um 10 02 58" src="https://user-images.githubusercontent.com/57712895/141955260-6c218e20-1ba6-43c4-8fcd-63152b354eed.png">

Reply:
that's sounds very wrong and must be an error in design. I see no reason for this to be different? What do you think?

Reply:
As discussed, I assumed that this was a mistake in the design and set the margin to 16px.